resources/views/site/home.blade.php

```diff
@@ -401,9 +453,6 @@ function toggleActive () {
 
 </script>
 <script src="{{ mix('js/home.js') }}"></script>
-@if(Carbon\Carbon::now()->month == 12 || Carbon\Carbon::now()->dayOfYear < 10)
```

Comment:
Are we getting rid of this?

Reply:
I think we should keep it seeing as its defiantly going to be used again next year...

Reply:
The implementation of the original snow (by me) was fairly quick and dirty. Pretty confident we could do something better.
Also fairly certain we will have a refreshed design by then... I hope!